lib/when_committed.rb

```diff
@@ -25,6 +34,10 @@ def run_when_committed_callbacks
     def clear_when_committed_callbacks
       when_committed_callbacks.clear
     end
+
+    def in_transcation?
```

Comment:
Keep in mind that all methods defined in the module will be inherited by objects that include that module. For that reason, it is good practice to keep the surface area of a mixin's interface very small.

It is possible that the including object already has an `in_transcation?` method (though, unlikely with that typo), defined by the author, or inherited by something like ActiveRecord. Its possible that a future version of ActiveRecord could define that method. And then objects that include this mixin would overwrite the definition, and things would break if the semantics are different.

I don't think the gain in code clarity is worth the potential trouble.